Sources/Core/AWSSDKIdentity/Sources/InternalAWSCommon/EmptyAWSCredentialIdentityResolver.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Smithy.Attributes
+import protocol SmithyIdentity.AWSCredentialIdentityResolver
+import struct SmithyIdentity.AWSCredentialIdentity
```

Comment:
This AWS credential identity resolver is used as the default value for AWS credential identity resolver in the "internal" clients.  Using this in place of the "default chain" resolvers eliminates the internal clients' dependence on AWSSDKIdentity.

Note that this client throws if you attempt to resolve credentials with it.  That's okay because the internal clients never actually use it; they either access "noAuth" operations or the identity resolver statically provides it credentials.